lib/msf/core/exploit/remote/auto_check.rb

```diff
@@ -57,11 +79,20 @@ def with_prepended_auto_check
       failure_type = Module::Failure::Unknown
     end
 
-    if datastore['ForceExploit']
-      print_warning("#{check_code.message} #{warning_msg}")
+    if auto_check_level <= AUTO_CHECK_LEVEL_BYPASSABLE && datastore['ForceExploit']
+      print_warning("#{check_code.message} ForceExploit is enabled, proceeding with exploitation.")
       return yield
     end
-    fail_with(failure_type, "#{check_code.message} #{error_msg}")
+
+    if auto_check_level <= AUTO_CHECK_LEVEL_BYPASSABLE
+      fail_with(failure_type, "#{check_code.message} Run \"set ForceExploit true\" to override check result.")
+    else
+      fail_with(failure_type, check_code.message)
+    end
+  end
+
+  def merge_info_autochecklevel(info, val)
+    info['AutoCheckLevel'] = val
```

Comment:
It looks like this method is not used and I cannot find a reference to the `info` hash.